src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Figure implements AreaCalculator, Drawable {

```

Comment:
This class should be `abstract`
```suggestion
public abstract class Figure implements AreaCalculator, Drawable {

```